lib/cron/startCron.js

```diff
@@ -68,6 +69,7 @@ var schedule = function(opts, callback) {
       log.info('The ' + opts.cronTask + ' task will run at ' + job.nextDates(1));
     }
 
+    job.start();
```

Comment:
@sandeepmjay  Just curious, can we set `start: true` in `new cron.CronJob(...)` and avoid the follow up above?

Reply:
One of the concerns was it would fire onTick immediately. But, looks like there is a separate flag runOnInit for that. It can be flipped. All constructor options are listed in the link.

https://github.com/kelektiv/node-cron